src/index.js

```diff
@@ -81,6 +101,13 @@ async function lambda(event) {
   }
 
   if (config.pullRequests.allowedActions.indexOf(requestBody.action) > -1) {
+    if (WEBHOOK_SECRET) {
+      const signature = event.headers[service.HTTP_SIGNATURE_HEADER] || null;
```

Comment:
What if this is added as a check before entering the if statement?
something like:
```js
if (WEBHOOK_SECRET && event.headers[service.HTTP_SIGNATURE_HEADER])
  const signature = event.headers[service.HTTP_SIGNATURE_HEADER];
```
From reading the docs, verify's signature is `Required`.

Reply:
Good call, fixed.